satpy/modifiers/atmosphere.py

```diff
@@ -77,6 +115,13 @@ def __call__(self, projectables, optional_datasets=None, **info):
             refl_cor_band = corrector.get_reflectance(sunz, satz, ssadiff,
                                                       vis.attrs['wavelength'][1],
                                                       red.data)
+
+        if lim_low > lim_high:
```

Comment:
If I understand it correctly you decided to drop the key `reduced_correction`, e.g. `reduced_correction: True` as you have in your PR description? And as far as I can see the default values will be `lim_low = 70`, `lim_high = 95` and `strength = 0`, meaning that this will only be become active if the user defines at least the `strength` in the YAML recipe?

 In that case I would add an if statement here to continue only if the strength is greater than 0. I guess the computations are not that heavy, but it might be confusing for the user the log says that Rayleigh is being reduced when they have not asked for it.

Reply:
I'll add that if.